ocs_ci/ocs/constants.py

```diff
@@ -180,6 +180,7 @@
 RIPSAW_DROP_CACHE = os.path.join(TEMPLATE_FIO_DIR, "drop_cache_pod.yaml")
 OCP_QE_DEVICEPATH_REPO = "https://github.com/anubhav-here/device-by-id-ocp.git"
 LOCAL_STORAGE_NAMESPACE = 'local-storage'
+NON_GA_LOCAL_STORAGE_NAMESPACE = 'openshift-local-storage'
```

Comment:
The variable name should be changed from ocp 4.6 onwards we are going to use `openshift-local-storage`

Reply:
We really shouldn't be changing the values of constants since that defeats the purpose of it being a constant. If the local storage namespace is variable based on the OCP version maybe we should move this to the default config and change that value based on the OCP version when we are deploying local storage. WDYT?

Reply:
i would prefer to move local storage namespace variable based on the OCP version to the default config

Reply:
we have already issue for new lso namespace #2640

Reply:
Swapped to using `openshift-local-storage` (regardless of OCP version) to address #2640